### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jakemannix**
cleaning up grammar!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 # dummyjvm
 Placeholder JVM-based repo
 
-Just settin' up my repo!
```
Comment (jakemannix) [inline README.md:4]: why bother wasting time with grammar?

**Review by jakemannix — COMMENTED** [review]
Please reconsider

File: README.md (hunk 1 above)
Comment (MattSamm) [reply README.md:4]: Because language is very powerful

Comment (tridenmitchell) [reply README.md:4]: It does look a little better

Comment (dasar208) [inline README.md:4]: Why exclamation point

**Comment by Billings007** [comment]
WOOOO

File: README.md (hunk 1 above)
Comment (lkeitadk) [reply README.md:4]: no need to bother with grammar when it makes sense !

Comment (dasar208) [inline README.md:4]: Request changes

**Review by dasar208 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jakemannix) [reply README.md:4]: better now?

**Review by dasar208 — APPROVED** [review]

**Pull request merged**